server/src/main/resources/configuration.yml

```diff
@@ -1,3 +1,6 @@
+
+components : ["Server", "Controller", "All"]
```

Comment:
I think we shouldn't use capitals unless they serve a purpose. 
I have been debating when to capitalize and when not to. So I am sticking to always smalls, unless I have a pretty good reason to keep something captialized. 

Otherwise it becomes a random mix of all small case and capital cases